apps/web/components/team/MemberListItem.tsx

```diff
@@ -147,6 +149,16 @@ export default function MemberListItem(props: Props) {
                     </Button>
                   </DropdownMenuItem>
                   <DropdownMenuSeparator className="h-px bg-gray-200" />
+                  <DropdownMenuItem>
+                    <Button
+                      onClick={() => setShowImpersonateModal(true)}
+                      color="minimal"
+                      StartIcon={LockClosedIcon}
+                      className="w-full flex-shrink-0 font-normal">
+                      {t("impersonate")}
+                    </Button>
+                  </DropdownMenuItem>
```

Comment:
Can we show this conditionally if the user has enabled impersonations? Or at least make it appears disabled.